bin/run.sh

```diff
@@ -223,27 +231,44 @@ tool_versions() {
     jq  --null-input \
         --arg gawk "$(gawk --version | head -1)" \
         --arg bats "$(bats --version)" \
-        --arg OS "$(. /etc/lsb-release; echo "$DISTRIB_DESCRIPTION")" \
+        --arg OS "$(
+                # probably don't need this complexity, but ...
+                if [[ -f /etc/lsb-release ]]; then
+                    . /etc/lsb-release; echo "$DISTRIB_DESCRIPTION"
+                elif command -v uname >/dev/null; then
+                    uname -sro
+                else
+                    echo "doesn't look like linux..."
```

Comment:
```suggestion
                    echo "Doesn't look like Linux..."
```

Reply:
The resulting .json file is not visible for public consumption. It just gets stashed in the db for later inspection.